main.ps1

```diff
@@ -91,7 +91,22 @@ Start-Process "chrome.exe" "https://office.com"
 Start-Process "chrome.exe" "https://teams.microsoft.com/v2" -Wait -PassThru
 
 # Easter egg ;)
-Invoke-WebRequest -Uri https://upload.wikimedia.org/wikipedia/commons/1/1f/Joe_Biden_81st_birthday.jpg -OutFile $env:USERPROFILE\Downloads\Joe_Biden_81st_birthday.jpg
-Start-Process $env:USERPROFILE\Downloads\Joe_Biden_81st_birthday.jpg
+$images = (Invoke-WebRequest "https://raw.githubusercontent.com/likes-gay/win-config/main/photos.txt").Content.Split([Environment]::NewLine)

```

Comment:
Can we use the link that will download the file from the releases rather than the repo

Reply:
Not sure, since maybe all releases should use the most recent version of the images

But with the `edge_redirect.ini`, maybe that should be from releases

Reply:
I have, but this has broken it more lol

Since the env which is supposed to get the current release version is only gonna be in the GitHub Actions, and not the the person's computer. Meaning it's gonna error now when it runs

Reply:
aint that yo whole thing

Reply:
guys we should have like a gang on gang rap battle between classes are you down

Reply:
we are kendrick lamar you are drake

Reply:
https://github.com/likes-gay/win-config/pull/1#issuecomment-2082864305